### PR TITLE
feat(spanner): add samples for creating instances with processing units

### PR DESCRIPTION
**Pull request by hengfengli**

### CODE REVIEW

**Review by skuruppu — APPROVED** [review]

**Comment by hengfengli** [comment]
Hi @dwsupplee, do you have any idea why the Lint check fails? It doesn't show me any error info.

**Comment by hengfengli** [comment]
My bad. `;` is missing at the end. 😞

**Comment by hengfengli** [comment]
@dwsupplee Can you please take a review? Thanks!

File: spanner/src/create_instance_with_processing_units.php [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+/**
+ * Creates an instance.
+ * Example:
+ * ```
+ * create_instance_with_processing_units($instanceId);
+ * ```
+ *
+ * @param string $instanceId The Spanner instance ID.
+ */
+function create_instance_with_processing_units($instanceId)
```
Comment (bshaffer) [inline spanner/src/create_instance_with_processing_units.php:38]: We should make it standard to use typehints / return types for all samples.
```suggestion
function create_instance_with_processing_units(string $instanceId): void
```

File: spanner/test/spannerTest.php [hunk 2]
```diff
@@ -45,6 +48,9 @@ class spannerTest extends TestCase
     /** @var $instance Instance */
     protected static $instance;
 
+    /** @var $instance lciInstance */
+    protected static $lciInstance;
```
Comment (bshaffer) [inline spanner/test/spannerTest.php:52]: sorry, but what does `lci` stand for? I'd appreciate a comment or a more descriptive name.

**Comment by hengfengli** [comment]
@bshaffer Please take a look again! Thanks.

**Comment by hengfengli** [comment]
@bshaffer Gentle ping.

**Review by bshaffer — APPROVED** [review]

**Pull request merged**